src/libs/Growl.ts

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import CONST from '../CONST';
+
+type GrowlRef = {
+    show?: (bodyText: string, type: string, duration: number) => void;
+};
+
+const growlRef = React.createRef<GrowlRef>();
+let resolveIsReadyPromise: undefined | ((value?: unknown) => void);
+const isReadyPromise = new Promise((resolve) => {
+    resolveIsReadyPromise = resolve;
+});
+
+function setIsReady(): void {
```

Comment:
```suggestion
function setIsReady() {
```

Reply:
Out of curiosity @fabioh8010, what's wrong with specifying `: void` for the return type? Is it more of a general best practice in TS, or a style preference? I can't find anything about that in https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md

Reply:
@francoisl  We discussed internally and we believe that using `void` return type is not adding any value altogether, so we decide to stop using, but we should include it in the style guide. I will set up a PR for that.

Reply:
Got it, thanks!